mne/report/report.py

```diff
@@ -2837,7 +2841,7 @@ def _add_raw(
             else:
                 fmax = np.inf
 
-            fig = raw.plot_psd(fmax=fmax, show=False, **add_psd)
+            fig = raw.compute_psd(fmax=fmax, **add_psd).plot(show=False, )
```

Comment:
> The kwargs used to be passed to plot_psd, now are passed to spectrum.plot(). Needs a deprecation cycle?

I'd say it's not worth it here given that the arguments are so similar. However, I do think you need to triage them, e.g., `tmin/tmin/fmin/fmax` need to go to `compute_psd` and plotting ones need to go to `.plot`. I don't think we want to remove any of these options, as people could reasonably want to just compute the PSD for the first 60 seconds for example, and say whether or not to plot in dB or whatever -- and these need to now go to two different methods.

Reply:
if we're going to triage then yeah, no need to deprecate. I'll pull out the triaging from the ToSpectrumMixin into a helper func so we can reuse it here.